web/app/src/model/game.ts

```diff
@@ -0,0 +1,45 @@
+import {z} from "zod";
+import {OnlinePlayer, zOnlinePlayer} from "./online-player";
+import {GamePlayer, zGamePlayer} from "./game-player";
+import {zCellContent} from "./cell";
+
+export const zGame = z.object({
+	uuid: z.string().uuid(),
+
+	board: z.array(z.array(zCellContent)),
+	currentPlayer: zGamePlayer,
+
+	players: z.array(zOnlinePlayer),
+});
+
+export type Game = z.infer<typeof zGame>;
+export type GameBoard = Game["board"];
+
+/**
+ * Find the green player in players list of the game.
+ * @param game
+ */
+export function getGameGreenPlayer(game: Game): OnlinePlayer {
+	return game.players.find(player => player.gamePlayer == GamePlayer.Green);
+}
+
+/**
+ * Find the red player in players list of the game.
+ * @param game
+ */
+export function getGameRedPlayer(game: Game): OnlinePlayer {
+	return game.players.find(player => player.gamePlayer == GamePlayer.Red);
+}
+
+/**
+ * Get the current online player.
+ * @param game The game from which to get the current player.
+ */
+export function getCurrentPlayer(game: Game): OnlinePlayer {
+	if (game.currentPlayer == GamePlayer.Green)
+		return getGameGreenPlayer(game);
+	else if (game.currentPlayer == GamePlayer.Red)
+		return getGameRedPlayer(game);
+	else
+		throw new Error("Unknown current player.");
+}
```

Comment:
Dunno why you need 3 functions while you could just do 

```suggestion
export function getCurrentPlayer(game: Game): OnlinePlayer {
		return game.players.find(player => player.gamePlayer == game.currentPlayer);
}
```